jetty-openid/src/main/java/org/eclipse/jetty/security/openid/OpenIdCredentials.java

```diff
@@ -224,40 +223,23 @@ and the Token Endpoint (which it is in this flow), the TLS server validation
         return paddedEncodedJwtSection;
     }
 
-    private Map<String, Object> claimAuthCode(String authCode) throws IOException
+    private Map<String, Object> claimAuthCode(HttpClient httpClient, String authCode) throws Throwable
```

Comment:
why make this exception type so general? probably not gonna throw `java.lang.AssertionError` e.g.

Reply:
I thought it might be a good idea to generalise it to indicate that you would need to handle other things like the IllegalArgumentExceptions. Have changed it back to just Exception for now.